proposals/0038-16bit-isspecialfloat.md

```diff
@@ -47,28 +58,29 @@ These are the existing Opcodes for the isSpecialFloat DXIL OpClass.
 9        | IsInf       | Returns true if x is +INF or -INF, false otherwise.
 10       | IsFinite    | Returns true if x is finite, false otherwise.
 
-* What DXIL changes does this change require?
-
 The following overload will be added:
 ```DXIL
 declare i1 @dx.op.isSpecialFloat.f16(
     i32,    ; opcode
     half)   ; val
 ```
 
-* How will SPIRV be supported?
+#### SPIRV changes
 
 There are the following SPIRV Ops:
 OpIsInf, OpIsNan, OpIsFinite.
 
 Currently the 32 bit version of these Ops are being used, but they
-have 16 bit versions as well.
+have 16 bit versions as well.  The open question is if we should always
+generate the 16 bit version of these ops, or if the generation should be
+gatekept behind some yet determined condition.
```

Comment:
We should have an open questions section at the end so these are all in one place and tracked, and I realize now that our template doesn't have one, so when I was saying to just add it as an open question I was assuming that there would be an obvious place to track these already.  Oops.  IMO we should add that to the template.